### PR TITLE
add `ray` modifier

### PR DESCRIPTION
**Pull request by edalzell**
Add support for Laravel Ray via the `ray` modifier.

Use like `dump`, for example, `{{ foo | ray }}`

### CODE REVIEW

File: src/Modifiers/CoreModifiers.php [hunk 1]
```diff
@@ -1496,6 +1496,17 @@ public function rawurlencode($value)
...
 
+    /**
+     * Send data to Laravel Ray.
+     *
+     * @param $value
+     * @return void
+     */
+    public function ray($value)
+    {
+        ray($value);
```
Comment (jasonvarga) [inline src/Modifiers/CoreModifiers.php:1507]: Return the `$value` so you can continue chaining modifiers.

**Review by jasonvarga — CHANGES_REQUESTED** [review]

**Comment by damcclean** [comment]
I've not tested this PR out but maybe it would be a good idea to throw an exception if the Ray package isn't actually installed?

**Comment by riasvdv** [comment]
Probably better to just check if the Ray class exists, and if not return the value so the modifier does nothing, wouldn't want it to crash just because you forgot a ray call somewhere

**Comment by jasonvarga** [comment]
A counter argument - if you add `| ray` to something and it does nothing, you might be thinking you're dumping in the wrong place. But maybe you just forgot to install it.

**Review by jasonvarga — APPROVED** [review]

**Comment by freekmurze** [comment]
Thank you! I'm honoured that support for Ray is now included by default in Statamic ❤️

**Pull request merged**